components/authentication/production/base/kustomization.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - ../../base/github
+components:
+  - ../k-components/rhtap-infra-secrets-patch
```

Comment:
what is the benefit of putting it in a component?

Reply:
that patch is used in both `production/base` and `production/stone-prod-p01`, I did not want to duplicate the patch

Reply:
hold on, I copied this from somewhere to share the patch but IIUC your comment, I could do the same with a resources. Let me test something

Reply:
If I convert the rhtap-infra-secrets-patch from a component to a patch, it does not work. The patch is applied before the resource to be patched is loaded.